src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift

```diff
@@ -195,10 +202,10 @@ enum AppStrings {
 		static let `internal` = NSLocalizedString("ExposureSubmissionError_internal", comment: "")
 		static let rateLimited = NSLocalizedString("ExposureSubmissionError_rateLimited", comment: "")
 		static let moreInfo = NSLocalizedString("ExposureSubmissionError_moreInfo", comment: "")
-		static let moreInfoURL = NSLocalizedString("ExposureSubmissionError_moreInfo_URL", comment: "")
-		static let moreInfoURLEN5 = NSLocalizedString("ExposureSubmissionError_moreInfo_URL_EN5", comment: "")
-		static let moreInfoURLEN11 = NSLocalizedString("ExposureSubmissionError_moreInfo_URL_EN11", comment: "")
-		static let moreInfoURLEN13 = NSLocalizedString("ExposureSubmissionError_moreInfo_URL_EN13", comment: "")
+		static let moreInfoURL = AppStrings.Links.appFaq
+		static let moreInfoURLEN5 = AppStrings.Links.appFaqENError5
+		static let moreInfoURLEN11 = AppStrings.Links.appFaqENError11
+		static let moreInfoURLEN13 = AppStrings.Links.appFaqENError13
```

Comment:
I left these be in order to not change too many unrelated files in this PR, but we can certainly adjust the references where they're used.

Reply:
I am going to adjust these in the Exposure Submission flow to get rid of the indirection.